_episodes/34.md

```diff
@@ -0,0 +1,296 @@
+     time: 1117
+   - title: "Pull requests of the episode"
+     desc: "Add Delta Lake connector and documentation."
+     time: 1570
+   - title: "Demo of the episode"
+     desc: "Delta Lake connector in action."
+     time: 1754
+   - title: "Question of the episode"
+     desc: "How do I secure the connection from a Trino cluster to the data source?"
+     time: 3240
+---
+
+## Guests
+
+In this episode Manfred has the pleasure to chat to two of our colleagues, who
+are working on making Trino better every day:
+
+* [Claudius Li](https://github.com/claudiusli), Product Manager at Starburst
+* [Joe Lodin](https://github.com/jhlodin), Information Engineer at Starburst
+
+Brian is out to add another member to his family!
+## Releases 372, 373, and 374
+
+Official notes from Martin Traverso:
+
+[Trino 372](https://trino.io/docs/current/release/release-372.html)
+
+* New `trim_array` function.
+* Support for reading ZSTD-compressed Avro files.
+* Support for column comments in Iceberg.
+* Support for Kerberos authentication in Kudu connector.
+
+[Trino 373](https://trino.io/docs/current/release/release-373.html)
+
+* New Delta Lake connector.
+* Improved performance of `LIKE` when querying Elasticsearch and PostgreSQL.
+* Improved performance when querying partitioned Hive tables.
+* Support access to S3 via HTTP proxy.
+
+[Trino 374](https://trino.io/docs/current/release/release-374.html)
+
+* Faster `GROUP BY` queries.
+* Vim/Emacs editing mode for CLI.
+* Support for `TRUNCATE TABLE` in Cassandra connector.
+* Support `uint` types in ClickHouse.
+* Support for Glue Metastore in Iceberg connector.
+* `CREATE/DROP SCHEMA`, table and column comments in MongoDB
+* Improved pushdown for PostgreSQL
+
+Notes from Manfred
+
+* Timeout configuration for LDAP authentication.
+* Values related to fault-tolerant execution in Web UI.
+* JDBC `Driver.getProperties` enables more client applications like DBVisualizer.
+* Vi and Emacs editing modes for interactive CLI usage.
+* Performance improvements in PostgreSQL connector.
+* SingleStore JDBC driver usage, end of `memsql` name.
+* Documentation for the atop connector.
+
+More detailed information is available in the
+[Trino 372](https://trino.io/docs/current/release/release-372.html),
+[Trino 373](https://trino.io/docs/current/release/release-373.html), and
+[Trino 374](https://trino.io/docs/current/release/release-374.html) release
+notes.
+
+## Project Tardigrade update
+
+The team around Project Tardigrade joined us in [episode 32](./32.html) to talk
+about fault tolerant execution of queries in Trino. Now they have posted a
+[status update on our blog]({%post_url 2022-02-16-tardigrade-project-update%}).
+
+It looks like things are really coming along well, and Joe has joined the effort
+to [create a first user facing documentation
+set](../docs/current/admin/fault-tolerant-execution.html).
+
+The team has also posted a status update on the Project Tardigrade slack
```

Comment:
uppercase Slack maybe use #project-tardigrade casing to match the Slack channel name